src/app.py

```diff
@@ -38,13 +54,34 @@ def zrp():
     # Build out a public facing API that responds to a get request with the full name, address, age, and gender as a query arguments. 
     # The API should return the probabilities of each race prediction as a JSON object.
     # Example of return object: {'AAPI': .56, 'Hispanic': .32, ..., 'White': .10}
+    # Required fields: 'Name_First', 'Name_Last', 'Name_Middle', 'Zipcode', 'Precinct_Split','Gender', 'County_Code','Congressional_District', 'Senate_District', 'House_District', 'Birth_Date'
 
     # As part of this, you should add any relevant pickle files that you might want to call to the directory "picklefiles".
     # These files will be stored on the server and your API will be able to call them in order to make predictions.
     # Currently, there's just one file in the picklefiles folder: the pickled Florida predictor that was sent over by Kasey, but feel free to add more.
     # Also, if you're having trouble accessing a picklefile in the picklefiles folder, DM Chris. It may be a problem with how the Dockerfile is set up
     # and how it copies the picklefile folder over to the container.
+
+    zipcode = request.args.get('zipcode')
+    last_name = request.args.get('last_name')
+    first_name = request.args.get('first_name')
+    middle_name = request.args.get('middle_name')
+    precinct_split = request.args.get('precinct_split')
+    gender = request.args.get('gender')
+    county_code = request.args.get('county_code')
+    congressional_district = request.args.get('congressional_district')
+    senate_district = request.args.get('senate_district')
+    house_district = request.args.get('house_district')
+    birth_date = request.args.get('birth_date')
+
+    data = {'Name_First': [str(first_name)], 'Name_Last': [str(last_name)], 'Name_Middle': [str(middle_name)], 'Zipcode': [int(zipcode)], 'Precinct_Split': [str(precinct_split)], 'Gender': [str(gender)], 'County_Code': [str(county_code)], 'Congressional_District':[float(congressional_district)], 'Senate_District': [float(senate_district)], 'House_District': [float(house_district)], 'Birth_Date': [str(birth_date)]}
```

Comment:
Style: could you split line 77 into several lines? Right now, it's a little difficult to read all on one line. Something like:

![image](https://user-images.githubusercontent.com/29349539/112421822-0f7e3800-8ced-11eb-8705-3af553e98533.png)

Reply:
Even though the statement takes up multiple lines, because it's all within one set of {}, Python doesn't throw an error.